pkgs/applications/science/astronomy/calcmysky/default.nix

```diff
@@ -20,7 +23,8 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [ cmake wrapQtAppsHook ];
-  buildInputs = [ glm eigen qtbase ];
+  buildInputs = [ glm eigen qtbase ]
+    ++ lib.optionals stdenv.isDarwin [ AGL AppKit OpenGL ];
```

Comment:
Or can we move these frameworks to `propagatedBuildInputs` of qtbase, saving us the trouble of specifying them in every downstream package?

Reply:
I felt like the Qt 5 maintainers [didn't really want to propagate them](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/libraries/qt-5/modules/qtbase.nix#L54). I also don't know to what extent Darwin frameworks should be propagated in qtbase...

Reply:
That's fine, just leave it as is, we can figure this out later.

Reply:
Now propagating inputs in order to build qtdeclarative and qtmultimedia.